.github/workflows/criterion_benchs.yml

```diff
@@ -1,10 +1,8 @@
-name: Benchmarks 
+name: CI
```

Comment:
why this change?

Reply:
To be consistent with the rest of the CI checks